README.md

```diff
@@ -1,60 +1,92 @@
-# Source Code Annotation application
+[![Build Status](https://travis-ci.org/src-d/code-annotation.svg)](https://travis-ci.org/src-d/code-annotation)
+![unstable](https://svg-badge.appspot.com/badge/stability/unstable?a)
+
+# Source Code Annotation Tool
 
 In order to evaluate quality of ML models, as well as to create “ImageNet for source core” there is a need for tools to automate the data collection/labeling/annotation.
 
+![Screenshot](.github/screenshot.png?raw=true)
+
 ## Installation
 
 ### Github OAuth tokens
 
-First you need OAuth application on github. [Read how to create it](https://developer.github.com/apps/building-oauth-apps/creating-an-oauth-app/).
+1. You need OAuth application on github. [Read how to create it](https://developer.github.com/apps/building-oauth-apps/creating-an-oauth-app/).
+
+    `Authorization callback URL: http://127.0.0.1:8080/oauth-callback`
 
-On a [page](https://github.com/settings/developers) with your application you will need `Client ID` and `Client Secret`
+2. Copy `.env.tpl` to `.env`.
 
-Copy `.env.tpl` to `.env` and set tokens there.
+3. On a [page](https://github.com/settings/developers) with your application find `Client ID` and `Client Secret` and put them in `.env` file.
 
 ### Docker
 
 ```bash
-docker build -t srcd/code-annotation .
-docker run --env-file .env --rm -p 8080:8080 srcd/code-annotation
+$ docker build -t srcd/code-annotation .
+$ docker run --env-file .env --rm -p 8080:8080 srcd/code-annotation
 ```
 
 ### Non-docker
 
 ```bash
-go get github.com/src-d/code-annotation/...
-cd $GOPATH/github.com/src-d/code-annotation
-make serve
+$ go get github.com/src-d/code-annotation/...
+$ cd $GOPATH/github.com/src-d/code-annotation
+$ make serve
 ```
 
-## Development
+## Importing and Exporting Data
 
-Backend:
+### Import File Pairs for Annotation
 
-```
-make gorun
-```
+The file pairs must be initially provided via an [SQLite](https://sqlite.org/) database. The database **must follow the expected schema**, please [follow this link](./cli/examples/import/example.sql) to see an example.
 
-Frontend:
+The `import` command will use those file pairs to create a new [SQLite](https://sqlite.org/) or [PostgreSQL](https://www.postgresql.org/) database that will be used internally by the Annotation Tool. The destination database does not need to be empty, new imported file pairs can be added to previous imports.
 
-If you want to benifit from frontend hot reloading feature this line in your `.env` file:
+_Please note_: if a file pair is identical to an existing one it will not be detected. A new pair entry will be created with the same contents.
```

Comment:
I'm not sure I understand this phrase.
Does this mean we can end up with duplicate results for files that are equal if we run more than the analysis more than once?

Please clarify

Reply:
@carlosms could you please handle it. It's a part about the import.

Reply:
To clarify: if a user logs in to annotate file pairs more than once, we will not end up with duplicate results. Previous annotation results are already saved for each user.

This only applies if you add new data to annotate to an already existing installation, using the "import" tool.

Let's say you decide to add new data, and one of those new file pairs is repoA repoB, src/A src/A.
If that very same file pair was existing from before, the DB will contain that previous pair, and the new one.

The only reason it works this way is because we decided not to guess too much. This can be changed easily when the ML team starts using the app and providing feedback.

Reply:
I find it a super-useful implementation-detail notice for the internal users and developer of the project.